add trusted contributor support

fries1234 opened the pull request:
this pr adds trusted contributor support to the bot and it improves the code a bit and it improves the formatting.

Commit:
add pycharm stuff

Commit:
run black linter

SomethingGeneric commented on bot.py:
```diff
@@ -1,50 +1,68 @@
                     if not issue.locked:
-                        issue.create_comment("This resolution has enough votes from Core Team mebers to be considered passed. Please close the issue once appropriate actions have been taken. - Beep Boop (I'm a bot, and this action was performed automagically)")
+                        issue.create_comment(
+                            "This resolution has enough votes from Core Team members to be considered passed. "
```
Nice work, can you just change this to "enough votes" :sweat_smile: then we'll send it!

fries1234 replied:
it seems like its already at "enough votes"

SomethingGeneric replied:
mb head empty i meant remove "from Core Team members" lol

fries1234 replied:
ok did that

Commit:
bot: add trusted contrib support and improved code

i formatted some of the code to make it a bit better and i added trusted
contributor support to the bot so it checks trusted contributors as well
as the core team.

SomethingGeneric commented:
Danke!
